code/perception/src/PositionPublisher.py

```diff
@@ -81,10 +74,8 @@ def update_current_pos(self):
         # x, y, z = self.transformer.gnss_to_xyz(lat, lon, alt)
 
         orientation_quat = self.current_pos_gps.pose.pose.orientation
-        self.current_heading = quat2heading([orientation_quat.x,
-                                            orientation_quat.y,
-                                            orientation_quat.z,
-                                            orientation_quat.w])[0]
+        self.current_heading = quat2heading(orientation_quat)
```

Comment:
```suggestion
        self.current_heading = quat_to_heading(orientation_quat)
```
Suggestion to be inline with current naming scheme.

Reply:
The import would also need to be updated.

Reply:
Fixed in 9dfd9398486344685369e5cc9cbd6258f74f01cf